src/sections/Home/Banner-4/banner4.style.js

```diff
@@ -2,6 +2,8 @@ import styled from "styled-components";
 
 const Banner1SectionWrapper = styled.section`
     display: none;
+    max-width: 1170px;
```

Comment:
please avoid just adding a random width. This will not fix the issue but will be overhead as maintaining. I am sure this will again have issues in future because we are not fixing the root issue but just silently fixing it. Use like some `width: 100%;` and then set max or min width based on responsiveness or use flex container styles please